streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -105,77 +105,27 @@ public void testNumProcesses() {
         final KStream<String, String> source2 = builder.stream(Arrays.asList("topic-3", "topic-4"), stringConsumed);
 
         final KStream<String, String> stream1 =
-            source1.filter(new Predicate<String, String>() {
-                @Override
-                public boolean test(final String key, final String value) {
-                    return true;
-                }
-            }).filterNot(new Predicate<String, String>() {
-                @Override
-                public boolean test(final String key, final String value) {
-                    return false;
-                }
-            });
+            source1.filter((key, value) -> true).filterNot((key, value) -> false);
```

Comment:
Should we keep one operator per line to make potential stack traces easier to read? Also simplifies setting breakpoint.

Reply:
ack